src/Classes/Config.php

```diff
@@ -253,12 +266,12 @@ final class Config
     public static $public_media_uri = '/media';
 
     /**
-     * The full web address to the image that will be served for a show if there
+     * The local path, relative to $public_media_path, to the image that will be served for a show if there
      * is not a photo for that show.
      *
      * @var string
      */
-    public static $default_show_uri = '/media/image_meta/ShowImageMetadata/22.png';
+    public static $default_show_path = 'image_meta/ShowImageMetadata/22.png';
```

Comment:
if we're doing this, mind changing this to have "image" or "photo" etc in the var name? :)